src/parse/ambient_lighting.c

```diff
@@ -0,0 +1,39 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ambient_lighting.c                                 :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: msloot <msloot@student.42.fr>              +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2025/05/30 21:45:35 by msloot            #+#    #+#             */
+/*   Updated: 2025/05/30 21:55:39 by msloot           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minirt.h"
+#include "parse.h"
+/*
+Ambient lighting:
+A 0.2 255,255,255
+  ∗ identifier: A
+  ∗ ambient lighting ratio in the range [0.0,1.0]: 0.2
+  ∗ (bonus) R,G,B colors in the range [0-255]: 10, 0, 255
+*/
+
+bool	parse_ambient_lighting(t_env *env, const t_line_arg *la)
+{
+	t_ambient	*al;
```

Comment:
```suggestion
	t_ambient	*a;
```
follow the letter notation